lib/active_shipping/carriers/usps.rb

```diff
@@ -371,12 +371,13 @@ def build_us_rate_request(packages, origin_zip, destination_zip, options = {})
     #
     # package.options[:mail_type] -- one of [:package, :postcard, :matter_for_the_blind, :envelope].
     #                                 Defaults to :package.
-    def build_world_rate_request(packages, destination, options)
+    def build_world_rate_request(origin, packages, destination, options)
       country = COUNTRY_NAME_CONVERSIONS[destination.country.code(:alpha2).value] || destination.country.name
       xml_builder = Nokogiri::XML::Builder.new do |xml|
         xml.IntlRateV2Request('USERID' => @options[:login]) do
           Array(packages).each_with_index do |package, id|
             xml.Package('ID' => id) do
+              xml.OriginZip(origin.zip)
```

Comment:
I know it is but if it doesn't break things if we include it everywhere, why not?

My experience with USPS APIs suggest that it should be okay to include it everywhere and it will just ignore it when it isn't needed. I'm not sure though, it would need to be tested.